src/System.IO.Pipelines/tests/BackpressureTests.cs

```diff
@@ -34,7 +34,7 @@ public void AdvanceThrowsIfFlushActiveAndNotConsumedPastThreshold()
             Assert.False(flushAsync.IsCompleted);
 
             ReadResult result = _pipe.Reader.ReadAsync().GetAwaiter().GetResult();
-            SequencePosition consumed = result.Buffer.GetPosition(result.Buffer.Start, 31);
+            SequencePosition consumed = result.Buffer.GetPosition(31, result.Buffer.Start);
```

Comment:
Change this to just specify the length without the position for the ones that pass start.